src/ipsis/course-merge/CourseMergeOfferingCollectionEntity.js

```diff
@@ -31,13 +31,17 @@ export class CourseMergeOfferingCollectionEntity extends BaseCollectionEntity {
 	}
 
 	selectedCourseMergeOfferingsHref() {
-		if (!this._entity.hasLinkByRel(Rels.ipsis.sisCourseMerge.selectedCourseMergeOfferings)) {
+		if (!this.hasCourseMergeOfferingsHref()) {
 			return;
 		}
 
 		return this._entity.getLinkByRel(Rels.ipsis.sisCourseMerge.selectedCourseMergeOfferings).href;
 	}
 
+	hasCourseMergeOfferingsHref() {
```

Comment:
Oops. Sorry. Noticed it when looking at the other PR. Should probably name this `hasSelectedCourseMergeOfferingsHref` to be more specific.

Reply:
Yes good catch